src/test/java/com/google/api/codegen/testdata/discoveries/py/py_deploymentmanager.v2.json.baseline

```diff
@@ -668,8 +668,8 @@ request = service.types().list(project=project)
 while request is not None:
     response = request.execute()
 
-    for type2 in response['types']:
-        # TODO: Change code below to process each `type2` resource:
-        pprint(type2)
+    for type_ in response['types']:
+        # TODO: Change code below to process each `type_` resource:
+        pprint(type_)
```

Comment:
@saicheems , you did the python MVVM refactoring for Discovery. Could you take a look to verify this doesn't break us?